client/components/promo-section/promo-card/index.tsx

```diff
@@ -91,6 +101,7 @@ const PromoCard: FunctionComponent< Props > = ( {
 			<ActionPanelBody>
 				{ title && (
 					<ActionPanelTitle className={ classNames( { 'is-primary': isPrimary } ) }>
+						{ icon && isCompact && <Gridicon icon={ icon } size={ 32 } /> }
```

Comment:
I'm only accounting for the existing of an icon here. May want to to also account for the existing of an image (see link 85 above).